collect_app/src/androidTest/java/org/odk/collect/android/support/pages/UserInterfacePage.java

```diff
@@ -24,15 +20,7 @@ public UserInterfacePage clickOnLanguage() {
     }
 
     public MainMenuPage clickOnSelectedLanguage(String language) {
-        try {
-            onView(withText(language)).perform(click());
-        } catch (NoMatchingViewException e) {
-            for (int i = 0; i < 10; i++) {
-                onView(withIndex(withId(android.R.id.text1), 1)).perform(ViewActions.swipeUp());
-            }
-            clickOnSelectedLanguage(language);
-        }
-
+        onData(equalTo(language)).perform(click());
```

Comment:
I changed this as I ran into flakes with `clickOnSelectedLanguage` while testing my changes to `killAndReopenApp`.